x-pack/plugins/security_solution/public/timelines/containers/index.gql_query.ts

```diff
@@ -210,6 +210,8 @@ export const timelineQuery = gql`
                   to
                   filters
                   note
+                  type
+                  threshold
```

Comment:
Ahh, here it is (in reference to https://github.com/elastic/kibana/pull/71371/files#r453934881) -- question still though, what features are gained by adding the additional fields to the timeline gql, is it just the `FieldBrowser`/`Event Details`, or more?